tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb

```diff
@@ -0,0 +1,4575 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        # and oll other länder
</pre><p>typo? (<code>all</code>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oscovida/oscovida/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Thanks, fixed.</p>